src/operator/rnn_impl.h

```diff
@@ -40,6 +40,9 @@
 #include "./mshadow_op.h"
 #include "./linalg.h"
 
```

Comment:
I missed this last time:
the file doesn't have name space. Please use namespace similar to rnn-inl.h

Reply:
fixed, thanks